cli/utils.go

```
@@ -223,3 +225,128 @@ func replaceLastOccurrence(search string, replacement string, subject string) st
+
+func looksLikeIpv6(uuid string) bool {
+	return ipv6Regex.MatchString(uuid)
+}
+
+func findGroupUuidByName(needle string, haystack Groups) (string, error) {
+	for _, group := range haystack {
+		if group.Name == needle {
+			return group.Id, nil
+		}
+	}
+	return "", errors.New(fmt.Sprintf("Couldn't find a group with name '%s'. Check the spelling.", needle))
```

golangci-lint (1.62.2)

S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

(gosimple)